ansible/roles/vm_set/tasks/add_ceos_list.yml

```diff
@@ -109,7 +109,7 @@
   become: yes
   docker_container:
     name: net_{{ vm_set_name }}_{{ vm_name }}
-    image: "{{ docker_registry_host }}/debian:jessie"
+    image: "{{ docker_registry_host }}/debian:bookworm"
```

Comment:
LGTM! If you haven't already could you please push the debian:bookworm image to our registry?

Reply:
Already done

Reply:
Hi @saiarcot895 
we should use  'docker pull debian:bookworm' get the latest one. Right?

Reply:
Hi @JibinBao, you shouldn't need to manually pull any container. It should automatically get pulled when running add-topo.

Reply:
Thanks